src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/adapters/TableAdapter.js

```diff
@@ -2,14 +2,29 @@
 import {observer} from 'mobx-react';
 import React from 'react';
 import Table from '../../../components/Table';
-import type {DatagridAdapterProps} from '../types';
+import AbstractAdapter from './AbstractAdapter';
 
 @observer
-export default class TableAdapter extends React.Component<DatagridAdapterProps> {
+export default class TableAdapter extends AbstractAdapter {
+    static getLoadingStrategy: () => string = () => { return 'pagination'; };
+    static getStorageStrategy: () => string = () => { return 'flat'; };
+
     static defaultProps = {
         data: [],
     };
 
+    renderCells(item: {[*]: *}, schemaKeys: Array<string>) {
+        return schemaKeys.map((schemaKey) => {
+            if (typeof item[schemaKey] === 'object') {
```

Comment:
Maybe add a todo comment here, saying that we have to find a proper solution for this.